### PR TITLE
build(all): build and publish es modules

### PR DESCRIPTION
**Pull request by fahrradflucht**

### CODE REVIEW

File: packages/demos/src/commonjs-module-loader/integrator.node.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import {FeatureAppManager, FeatureServiceRegistry} from '@feature-hub/core';
-import {loadCommonJsModule} from '@feature-hub/module-loader/lib/node';
+import {loadCommonJsModule} from '@feature-hub/module-loader/lib/cjs/node';
```
Comment (clebert) [inline packages/demos/src/commonjs-module-loader/integrator.node.tsx]: do we also refer to cjs in the esm build?

Comment (fahrradflucht) [reply packages/demos/src/commonjs-module-loader/integrator.node.tsx]: I'm not sure I understand. This import is in a demo which doesn't get built to esms.

File: jest.config.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ module.exports = {
   moduleFileExtensions: ['js', 'json', 'ts', 'tsx'],
   moduleNameMapper: {
     '^@feature-hub/([^/]+)$': '<rootDir>/packages/$1/src/index.ts',
-    '^@feature-hub/module-loader/lib/node$':
+    '^@feature-hub/module-loader/lib/cjs/node$':
```
Comment (clebert) [inline jest.config.js]: add esm mapping

File: tsconfig.json [hunk 3]
```diff
@@ -19,7 +19,7 @@
     "baseUrl": ".",
     "paths": {
       "@feature-hub/*": ["packages/*/src/index.ts"],
-      "@feature-hub/module-loader/lib/node": [
+      "@feature-hub/module-loader/lib/cjs/node": [
```
Comment (clebert) [inline tsconfig.json]: add esm mapping

**Pull request merged**